### PR TITLE
Fix chart dialog styles

### PR DESCRIPTION
**Pull request by korvin89**
<details>
<summary>Before</summary>

image

</details>
<details>
<summary>After</summary>

image

</details>

[CHARTS-8363]

### CODE REVIEW

File: src/ui/units/dash/containers/DropdownNavigation/DropdownNavigation.js [hunk 1]
```diff
@@ -185,6 +151,41 @@ class DropdownNavigation extends React.PureComponent {
...
+                    onClick={() => this.setState({showNavigation: !this.state.showNavigation})}
+                    ref={this.setButtonRef}
+                    className={b('button')}
+                >
+                    {this.state.entry ? (
+                        <EntryTitle entry={this.state.entry} theme="inline" />
+                    ) : (
+                        i18n('dash.navigation-input.edit', 'button_choose')
+                    )}
+                    {/* <Icon
```
Comment (Marginy605) [inline src/ui/units/dash/containers/DropdownNavigation/DropdownNavigation.js:180]: Let's remove commented code

File: src/ui/units/dash/components/NavigationInput/NavigationInput.scss [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
         display: grid;
-        grid-template-columns: var(--dl-dropdown-navigation-width) 75px 75px;
+        grid-template-columns: var(--dl-dropdown-navigation-width) auto auto;
         grid-column-gap: 10px;
         align-items: center;
     }
 
     &__button {
-        width: 135px;
+        width: fit-content;
```
Comment (Marginy605) [inline src/ui/units/dash/components/NavigationInput/NavigationInput.scss:10]: Button changes it's width after click on "Specify link" below. Looks weird.

**Review by Marginy605 — CHANGES_REQUESTED** [review]

**Comment by mournfulCoroner** [comment]
This is a rare user case, but could it be better to stretch the width of these two buttons?


<details>
<summary>Example</summary>

image

image

</details>

**Comment by korvin89** [comment]
>   This is a rare user case, but could it be better to stretch the width of the "Open" button to the width of the two buttons at the bottom?
> 
> Example

I think it looks a little bit weird

**Review by mournfulCoroner — APPROVED** [review]

**Review by Marginy605 — APPROVED** [review]

**Pull request merged**